edocumentation-domain/src/main/java/org/dcsa/edocumentation/domain/persistence/entity/ShippingInstruction.java

```diff
@@ -9,11 +9,15 @@
 
 import jakarta.validation.Valid;
 import jakarta.validation.Validator;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Null;
 import lombok.*;
 import org.dcsa.edocumentation.domain.dfa.*;
```

Comment:
i am not sure * is the standard way of importing ?

Reply:
Ok.. point to be noted. I will approve the pull request. I will forward this point to Jeroen.